pom.xml

```diff
@@ -46,29 +46,28 @@
 
         <!-- Plugins -->
         <version.microprofile.build-tools>1.2</version.microprofile.build-tools>
-
-        <version.plugin.compiler>3.10.1</version.plugin.compiler>
-        <version.plugin.resources>3.2.0</version.plugin.resources>
-        <version.plugin.jar>3.2.0</version.plugin.jar>
-        <version.plugin.dependency>3.1.2</version.plugin.dependency>
-        <version.plugin.source>3.2.0</version.plugin.source>
-        <version.plugin.javadoc>3.2.0</version.plugin.javadoc>
-
-        <version.plugin.helper>3.3.0</version.plugin.helper>
-        <version.plugin.impsort>1.8.0</version.plugin.impsort>
+        <version.plugin.compiler>3.13.0</version.plugin.compiler>
+        <version.plugin.resources>3.3.1</version.plugin.resources>
+        <version.plugin.jar>3.4.1</version.plugin.jar>
+        <version.plugin.dependency>3.7.0</version.plugin.dependency>
+        <version.plugin.source>3.3.1</version.plugin.source>
+        <version.plugin.javadoc>3.7.0</version.plugin.javadoc>
+        <version.plugin.helper>3.6.0</version.plugin.helper>
+        <version.plugin.impsort>1.10.0</version.plugin.impsort>
         <version.plugin.formatter>2.22.0</version.plugin.formatter>
+        <version.plugin.checkstyle>3.4.0</version.plugin.checkstyle>
+        <version.plugin.rat>0.16.1</version.plugin.rat>
+        <version.plugin.bnd>7.0.0</version.plugin.bnd>
```

Comment:
BND 7 requires Java 17.

With impsort, formatter and bnd reverted to their old versions, I can build this project and the MP OpenAPI spec with Java 11.

Reply:
Let's update the build to use Java 17 to build as the previous version of impsort has CVEs.